Controllers/ApiController.cs

```diff
@@ -43,68 +43,24 @@ public ActionResult<string> GetPaymentMethods([FromBody] PaymentMethodsRequest r
             }
         }
 
-        [HttpPost("api/initiatePayment")]
-        public ActionResult<string> InitiatePayment([FromBody] PaymentRequest pmreq)
+        [HttpPost("api/sessions")]
+        public ActionResult<string> Sessions()
         {
-            _logger.LogInformation($"Request for Payments API::\n{pmreq}\n");
+            var sessionsRequest = new CreateCheckoutSessionRequest();
+            sessionsRequest.merchantAccount = _merchant_account; // required
+            sessionsRequest.channel = (CreateCheckoutSessionRequest.ChannelEnum?) PaymentRequest.ChannelEnum.Web; // required
```

Comment:
Is `channel` really required?

Reply:
It is not, good catch. [Reference](https://docs.adyen.com/api-explorer/#/CheckoutService/v68/post/sessions__reqParam_channel)